tests/components/ecobee/test_repairs.py

```diff
@@ -77,3 +83,60 @@ async def test_ecobee_repair_flow(
         issue_id="migrate_notify",
     )
     assert len(issue_registry.issues) == 0
+
+
+async def test_ecobee_aux_heat_repair_flow(
+    hass: HomeAssistant,
+    mock_ecobee: MagicMock,
+    hass_client: ClientSessionGenerator,
+    issue_registry: ir.IssueRegistry,
+) -> None:
+    """Test the ecobee aux_heat service repair flow is triggered."""
+    await setup_platform(hass, CLIMATE_DOMAIN)
+    await async_process_repairs_platforms(hass)
+
+    http_client = await hass_client()
+
+    ENTITY_ID = "climate.ecobee2"
+
+    # Simulate legacy service being used
+    assert hass.services.has_service(CLIMATE_DOMAIN, SERVICE_SET_AUX_HEAT)
+    await hass.services.async_call(
+        CLIMATE_DOMAIN,
+        SERVICE_SET_AUX_HEAT,
+        {ATTR_ENTITY_ID: ENTITY_ID, ATTR_AUX_HEAT: True},
+        blocking=True,
+    )
+    await hass.async_block_till_done()
+
+    # Assert the issue is present
+    assert issue_registry.async_get_issue(
+        domain="ecobee",
+        issue_id="migrate_aux_heat",
+    )
+    assert len(issue_registry.issues) == 1
+
+    url = RepairsFlowIndexView.url
+    resp = await http_client.post(
+        url, json={"handler": "ecobee", "issue_id": "migrate_aux_heat"}
+    )
+    assert resp.status == HTTPStatus.OK
+    data = await resp.json()
+
+    flow_id = data["flow_id"]
+    assert data["step_id"] == "confirm"
+
+    url = RepairsFlowResourceView.url.format(flow_id=flow_id)
+    resp = await http_client.post(url)
+    assert resp.status == HTTPStatus.OK
+    data = await resp.json()
+    assert data["type"] == "create_entry"
+    # Test confirm step in repair flow
+    await hass.async_block_till_done()
+
+    # Assert the issue is no longer present
+    assert not issue_registry.async_get_issue(
+        domain="ecobee",
+        issue_id="migrate_aux_heat",
+    )
+    assert len(issue_registry.issues) == 0
```

Comment:
This integration uses a standard repair with no custom flows. There is no need to test this part for that reason (that responsibility is already satisfied by the repairs integration itself).

```suggestion
```